From 3bacce956692ebd2be1b9d81d0358ea438ebf6cf Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 11 Jan 2024 09:32:51 +0100
Subject: [PATCH 0444/1695] pta: scmi: remove noisy info level message on
 message process

Remove useless trace message emitted each time SCP-firmware
successfully processes an SCMI message.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b8192642e8f5400905818ca05e3b625dd1b4e58a pta: scmi: remove noisy info level message on message process]
Change-Id: I977e41af3fb1c0591a82152c68b40f5f14a275b3
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/pta/scmi.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/core/pta/scmi.c b/core/pta/scmi.c
index a0efcc073..54f822133 100644
--- a/core/pta/scmi.c
+++ b/core/pta/scmi.c
@@ -170,10 +170,8 @@ static TEE_Result cmd_process_msg_channel(uint32_t ptypes,
 		res = scmi_server_msg_process_thread(channel_id, in_buf,
 						     in_size, out_buf,
 						     &out_size);
-		if (!res) {
+		if (!res)
 			params[2].memref.size = (uint32_t)out_size;
-			IMSG("scmi optee shm: out %zu", out_size);
-		}
 
 		return res;
 	}
-- 
2.47.2

